src/lints/enum_no_repr_variant_discriminant_changed.ron

```diff
@@ -0,0 +1,78 @@
+SemverQuery(
+    id: "enum_no_repr_variant_discriminant_changed",
```

Comment:
Naming is hard. This is not the best name, and not the worst. Nothing better comes to my mind, but if you have other suggestions, it would be good to consider them.